ros_cross_compile/docker/build_workspace.sh

```diff
@@ -20,8 +21,7 @@ fi
 set -ux
 colcon build --mixin "${TARGET_ARCH}"-docker \
   --build-base build_"${TARGET_ARCH}" \
-  --install-base install_"${TARGET_ARCH}" \
-  --event-handlers console_cohesion+ console_package_list+
+  --install-base install_"${TARGET_ARCH}"
```

Comment:
Can you think of a good way to default to these handlers without forcing them? Maybe a preinstalled `defaults.yaml` file that does this, if the user doesn't provide their own file? I would still like this format to be the default, but I agree that it shouldn't be forced.

Reply:
Yep, added in `sysroot_creator.py`.